LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan.NetworkServer/DownlinkMessageBuilder.cs

```diff
@@ -79,21 +79,21 @@ internal static class DownlinkMessageBuilder
 
             if (receiveWindow is ReceiveWindow2)
             {
-                freq = loRaRegion.GetDownstreamRX2Freq(configuration.Rx2Frequency, logger, deviceJoinInfo);
```

Comment:
[nit] This is technically related to bug #1319 but no need to revert since this will change anyway

Reply:
Agree, needed to fix it with the minimum changes possible since it broke my integration test.